config/locales/en.yml

```diff
@@ -326,3 +326,11 @@ en:
               min_grade:          "must be greater than 0"
             user:
               blank:              "can't be blank'"
+        assignment:
+          attributes:
+            user:
+              blank:                 "User can't be null"
+            offspring:
+              blank:                 "must exist"
```

Comment:
can't be blank, do it with an standard message and allow the customer to update it changing the yml file